packages/@okta/vuepress-site/docs/release-notes/2021-okta-identity-engine/index.md

```diff
@@ -1,12 +1,27 @@
 ---
-title: Okta Identity Engine API Products Release Notes 2021
+title: Okta Identity Engine API Products release notes 2021
 ---
 <ApiLifecycle access="ie" /><br>
 <ApiLifecycle access="Limited GA" /><br>
 
 ## November
 
-### Weekly Release 2021.11.2
+### Weekly release 2021.11.3
+
+| Change                                                                   | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Support for multiple device management statuses on a single device is now GA in Preview](#support-for-multiple-device-management-status-on-a-single-device-is-now-ga-in-preview) | December 02, 2021 |
+| [Bug fixed in 2021.11.3](#bug-fixed-in-2021-11-3) | December 02, 2021 |
+
+#### Support for multiple device management statuses on a single device is now GA in Preview
+
+Multiple users signed in to a single device now retain their own device management status. Users with a `managed` device status don't share that status with users who have an `unmanaged` device status. See [List devices (expand=user)](/docs/reference/api/devices/#usage-example-expand-user).<!--OKTA-447458-->
```

Comment:
The second line sounds like the users are the actors "sharing" their device status with other users. From the example, it looks like the response from the `expand-user` option provides both `managed` and `unmanaged` status for each user? This is just a question
The PM approved, so this blurb should be ok.

Reply:
```suggestion
Multiple users signed in to a single device now retain their own device management status. Users with a `managed` device status don't share that status with users who have an `unmanaged` device status.<!--OKTA-447458-->
```